shared/rust/src/domain/asset.rs

```diff
@@ -124,6 +124,16 @@ pub enum AssetId {
     ResourceId(ResourceId),
 }
 
+impl Display for AssetId {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::JigId(jig_id) => write!(f, "{}", &jig_id.0),
+            Self::CourseId(course_id) => write!(f, "{}", &course_id.0),
+            Self::ResourceId(resource_id) => write!(f, "{}", &resource_id.0),
+        }
+    }
+}
+
 impl From<JigId> for AssetId {
```

Comment:
I wouldn't impl display on something that's not meant to be displayed.

I'd rather use `.uuid().to_string()`, add a `.to_string()` method, or use `debug`.

Let me know if you disagree

Reply:
I missed the `.uuid()` fn